models/tests.go

```diff
@@ -0,0 +1,103 @@
+// Copyright 2016 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package models
+
+import (
+	"github.com/go-xorm/core"
+	"github.com/go-xorm/xorm"
+	_ "github.com/mattn/go-sqlite3" // for the test engine
```

Comment:
I'd keep SQLite for now, until I can add support to TiDB on testfixtures.

@lunny What do you think?